oxli/khmer_api.py

```diff
@@ -0,0 +1,31 @@
+#
+# This file is part of khmer, http://github.com/ged-lab/khmer/, and is
+# Copyright (C) Michigan State University, 2009-2015. It is licensed under
+# the three-clause BSD license; see doc/LICENSE.txt.
+# Contact: khmer-project@idyll.org
+#
+# pylint: disable=missing-docstring
+
+
+import threading
+import khmer.utils
+
+
+def build_graph(ifilenames, graph, num_threads=1, tags=False):
```

Comment:
naming inputs with a 'i' prefix is not a convention we use. Also needs docstrings.